plugins/modules/zos_stat.py

```diff
@@ -0,0 +1,1685 @@
+        password:
+          description:
+            - Whether the data set has a password set to read/write.
+            - Value can be either one of 'NONE', 'READ' or 'WRITE'.
+            - For VSAMs, the value can also be 'SUPP', when the module
+              is unable to query its security attributes.
+          returned: success
+          type: str
+          sample: NONE
+        racf:
+          description:
+            - Whether there is RACF protection set on the data set.
+            - Value can be either one of 'NONE', 'GENERIC' or 'DISCRETE'
+              for non-VSAM data sets.
+            - For VSAMs, the value can be either 'YES' or 'NO'.
+          returned: success
+          type: str
+          sample: NONE
+        key_label:
+          description: The encryption key label for an encrypted data set.
+          returned: success
+          type: str
+          sample: KEYDSN
+        dir_blocks_allocated:
+          description:
+            - Number of directory blocks allocated for a PDS.
+            - For PDSEs, this value will be null. See instead pages_used
+              and perc_pages_used.
+          returned: success
+          type: int
+          sample: 5
+        dir_blocks_used:
+          description:
+            - Number of directory blocks used by a PDS.
+            - For PDSEs, this value will be null. See instead pages_used
+              and perc_pages_used.
+          returned: success
+          type: int
+          sample: 2
+        members:
+          description: Number of members inside a partitioned data set.
+          returned: success
+          type: int
+          sample: 3
+        pages_allocated:
+          description: Number of pages allocated to a PDSE.
+          returned: success
+          type: int
+          sample: 1116
+        pages_used:
+          description:
+            - Number of pages used by a PDSE. The pages are 4K in size.
+          returned: success
+          type: int
+          sample: 5
+        perc_pages_used:
+          description:
+            - Percentage of pages used by a PDSE.
+            - Gets rounded down to the nearest integer value.
+          returned: success
+          type: int
+          sample: 10
+        pdse_version:
+          description: PDSE data set version.
+          returned: success
+          type: int
+          sample: 1
+        max_pdse_generation:
+          description:
+            - Maximum number of generations of a member that can be
+              maintained in a PDSE.
+          returned: success
+          type: int
+          sample: 0
+        seq_type:
+          description:
+            - Type of sequential data set (when it applies).
+            - Value can be either one of 'BASIC', 'LARGE' or 'EXTENDED'.
+          returned: success
+          type: str
+          sample: BASIC
+        data:
+          description:
+            - Dictionary containing attributes for the DATA part of a VSAM. 
```

Comment:
In VSAM these parts are called components.
```suggestion
            - Dictionary containing attributes for the DATA component of a VSAM. 
```